jhipster-jdl.jdl

```diff
@@ -15,7 +15,7 @@ enum AlterationType {
 }
 
 enum ArticleType {
-    PMID, ABSTRACT
+    PMID, ABSTRACT, REFERENCE, FDADRUG_LETTER, FDADRUG_LABEL, FDADRUG_SUMMARY, FDADRUG_SUMMARY_REVIEW
```

Comment:
No longer user a prefix for PMID, ABSTRACT, and REFERENCE?

Reply:
I thought about it, but these three are more like categories. on the same level of FDADRUG